src/Files.App/Data/Items/ListedItem.cs

```diff
@@ -326,6 +328,30 @@ public ObservableCollection<FileProperty> ItemProperties
 			set => SetProperty(ref itemProperties, value);
 		}
 
+		private int imageWidth;
+		public int ImageWidth
+		{
+			get => imageWidth;
+			set
+			{
+				SetProperty(ref imageWidth, value);
+				OnPropertyChanged(nameof(DimensionsDisplay));
+			}
+		}
+
+		private int imageHeight;
+		public int ImageHeight
+		{
+			get => imageHeight;
+			set
+			{
+				SetProperty(ref imageHeight, value);
+				OnPropertyChanged(nameof(DimensionsDisplay));
+			}
+		}
+
+		public string DimensionsDisplay => IsImage && ImageWidth > 0 && ImageHeight > 0 ? $"{ImageWidth} \u00D7 {ImageHeight}" : string.Empty;
```

Comment:
Is there a way to lazy load this property so that it only loads the first time we try retrieving it?

Reply:
1. That’s what I’m actually thinking about in my Storage Layer concept for properties. But is this loaded more than once?
2. If we should we should make more flexible extension methods. Such as IsCompatibleToSetAsWallpaper and IsCompatibleToGetDimentions.
3. I thought the same but I saw this used in MSDN. https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/m/multiplication-sign

Reply:
1. It's about only loading it when it's needed, otherwise we're spending compute time on a property that'll only be displayed on hover.
2. I think that we can keep `IsImage` as a general extension method for image files, and we can create a more specific one for `IsCompatibleToSetAsWallpaper`.
3. Character Map UWP has a feature to filter out deprecated icons, it might be worth checking there.

Reply:
All done.

Reply:
Testing rn.

Reply:
> I think the icon u00D7 is deprecated, can you try uE711 instead?

It is garbling.

![image](https://github.com/files-community/Files/assets/62196528/ed29d247-6bd6-441a-bc32-f9b571d90ba1)

Reply:
I'll check into the icon, as far as the first point, do you have any ideas on if this can be done?

Reply:
First point means *Is there a way to lazy load this property so that it only loads the first time we try retrieving it?*?

I've implemented this using a property. Property doesn't prepare its value until actually requested to obtain.

Reply:
It shouldn't be an icon, but the Multiply glyph
<img width="536" alt="image" src="https://github.com/files-community/Files/assets/7389110/81bd65ce-279c-449f-b732-9d0cbf31253a">